src/types/near_token.rs

```diff
@@ -11,6 +11,7 @@ const ONE_NEAR: u128 = 10u128.pow(24);
     derive_more::From,
     derive_more::Into,
     derive_more::FromStr,
+    PartialEq,
```

Comment:
Move it up (after `Copy`), and add `Eq`, `PartialOrd`, and `Ord` derives for completeness.

Reply:
Done